build.xml

```diff
@@ -42,6 +42,17 @@
       <arg value="-no-header"/>
       <arg value="examples/gsafeed.dtd"/>
     </exec>
+    <exec executable="xjc">
```

Comment:
We are submitting generated classes too, yes? When are the classes generated?  Or is this a reference for posterity?

Reply:
Gennis included this target in the first CL. It both documents what we used to generate the classes and could be used to generate them again for comparison with the code we're planning to modify. If we end up not using this, or the diffs become unusable, we can just document this somewhere in a README or something.